config/initializers/constants.rb

```diff
@@ -0,0 +1,11 @@
+module QuoraClone
+  EMAIL_REGEX = /\A(|(([A-Za-z0-9]+_+)|([A-Za-z0-9]+\-+)|([A-Za-z0-9]+\.+)|([A-Za-z0-9]+\++))*[A-Za-z0-9]+@((\w+\-+)|(\w+\.))*\w{1,63}\.[a-zA-Z]{2,6})\z/i
```

Comment:
why making modules here
use all constants in global namespace

Reply:
module QuoraClone
  module RegexConstants
    EMAIL_REGEX =//
 end
end